### PR TITLE
Setting custom height to modal through js open() property

### PR DESCRIPTION
**Pull request by GabrielHobold**
Added a height property to open() method. It's now possible to set a custom height without creating a css class.
Saw the issue at [this stack overflow post](http://stackoverflow.com/questions/37903111/how-to-change-the-height-of-ngdialog).

### CODE REVIEW

**Comment by GabrielHobold** [comment]
Check build errors.

File: tests/protractor/open.js [hunk 1]
```diff
@@ -19,4 +19,9 @@ describe('ngDialog open', function() {
     element(by.css('#js-width')).click();
     expect(element(by.css('.ngdialog-content')).getCssValue('width')).toBe('650px');
   });
+
+    it('should define custom height through a js property', function() {
```
Comment (faceleg) [inline tests/protractor/open.js]: Inconsistent indentation

**Comment by faceleg** [comment]
Looks good, great that you updated the README.

Could you add a test that proves the dialog still works if `height` is omitted entirely?

**Comment by GabrielHobold** [comment]
Indentation corrected.
Modal should have height even if the `height` property is omitted.

**Comment by faceleg** [comment]
Thank you very much!

**Pull request merged**